packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/lib_auth/data_sources/remote/auth_data_source.g.dart

```diff
@@ -63,6 +63,28 @@ class _AuthDataSource implements AuthDataSource {
     return null;
```

Comment:
Please do not commit generated files. They are now generated during as part of the `rx_bloc_cli create` command
![Screenshot 2023-03-06 at 11 01 20](https://user-images.githubusercontent.com/32831935/223157919-639f1de3-e900-4b91-83eb-7439cf463a46.png)